auth/saml.adoc

```diff
@@ -226,3 +227,67 @@ with enough administrative privileges to update it:
 * ssh to the appliance as root
 * Run _appliance_console_cli --extauth-opts_ *local_login_disabled=false*
 
+[[configuring-saml-via-cli]]
+== Configuring SAML using the CLI
+
+Another way to configure SAML on the appliance is to use the appliance
+console CLI instead of the above steps. The `--saml-config` subcommand
+of the appliance console CLI allows one to configure the appliance for
+SAML Authentication by updating both Apache configurations as well as the
+necessary Administrative UI settings. The `--saml-unconfig` subcommand
+is also provided for unconfiguring the appliance SAML Authentication
+and reverting the appliance to Database Authentication.
+
+Usage of the new Appliance Console CLI subcommands are as follows:
+
+=== Configure SAML
+To configure the appliance for SAML Authentication:
+
+```
+# appliance_console_cli --saml-config [--saml-client-host=miq_appliance.fqdn] \
+                                       --saml-idp-metadata=file|url           \
+                                      [--saml-enable-sso]
+```
+
+When --saml-client-host is not specified, the configured appliance host
+is used for creating the SP metadata. This host fqdn must be reachable from
+the SAML IDP.
+
+Configuring SAML requires the IDP metadata file, the --saml-idp-metadata option allows
+the user to specify a copy by file path that was downloaded to the appliance or by URL
+in which case the CLI downloads the copy from the IDP.
```

Comment:
This wording seems odd to me. May I suggest you consider:

Configuring SAML requires the IDP metadata file. The --saml-idp-metadata option allows
the user to specify the file path of a copy that was downloaded to the appliance or by a URL,
in which case the CLI downloads the copy from the IDP.